gamemode/modules/base/sh_util.lua

```diff
@@ -97,23 +97,25 @@ function DarkRP.findPlayers(info)
     local InfoPlayers = {}
     for A in string.gmatch(info .. ";", "([a-zA-Z0-9:_.]*)[;(,%s)%c]") do
         if A ~= "" then
-            table.insert(InfoPlayers, A)
+            InfoPlayers[#InfoPlayers+1] = A
         end
     end
 
-    for _, PlayerInfo in ipairs(InfoPlayers) do
```

Comment:
Same question with `ipairs`. The use of `ipairs` over `pairs` already is because it's faster